wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -21,52 +21,33 @@ import CoreLocation
 import Contacts
 import WireDataModel
 import WireCommonComponents
+import SwiftUI
 
 class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
 
     // MARK: - Properties
     typealias LabelColors = SemanticColors.Label
+    private typealias deviceDetailsSection = L10n.Localizable.Device.Details.Section
+
+    let proteusVerficiationStatusImageView = UIImageView()
 
     let nameLabel = DynamicFontLabel(fontSpec: .normalSemiboldFont,
                                      color: LabelColors.textDefault)
     let labelLabel = DynamicFontLabel(fontSpec: .smallSemiboldFont,
                                       color: LabelColors.textDefault)
-    let activationLabel = UILabel(frame: CGRect.zero)
-    let fingerprintLabel = UILabel(frame: CGRect.zero)
-    let verifiedLabel = DynamicFontLabel(fontSpec: .smallFont,
-                                         color: LabelColors.textDefault)
-
-    private let activationLabelFont = FontSpec.smallLightFont
-    private let activationLabelDateFont = FontSpec.smallSemiboldFont
+    let mlsThumbprintLabel = DynamicFontLabel(fontSpec: .mediumRegularFont,
+                                        color: LabelColors.textCellSubtitle)
+    let proteusIdLabel = DynamicFontLabel(fontSpec: .mediumRegularFont,
+                                            color: LabelColors.textCellSubtitle)
 
-    var showVerified: Bool = false {
-        didSet {
-            updateVerifiedLabel()
-        }
-    }
+    private let verifiedImage =  Asset.Images.verifiedShield.image.resizableImage(withCapInsets: .zero)
```

Comment:
question: Are we sure the color of the image updates appropriately when switching from light to dark and vice versa?

Reply:
done. It works